src/hals.jl

```diff
@@ -20,7 +20,7 @@ function update!(data, W, H, meta, options)
 
     # H update
     _setup_H_update!(W, H, meta)
-    if get(options, "mode", nothing) == "parallel"
+    if get(kwargs, :mode, nothing) == "parallel"
```

Comment:
Does get work this way with `kwargs`? I was under the impression that `kwargs` was no longer a dictionary.